.github/workflows/prod-auto-tag-and-release.yml

```
@@ -0,0 +1,40 @@
+name: Auto Tag and Release
+
+on:
+  push:
+    branches:
+      - main
```

`pg-to-dbml` was prior to the Github rollout of the new main branch name.

```suggestion
      - master
```